wire-ios/Wire-iOS/Resources/Base.lproj/Localizable.strings

```diff
@@ -305,6 +305,7 @@
 "conversation.create.guidance.toolong" = "Too many characters";
 "conversation.create.options.title" = "Conversation options";
 "conversation.create.options.subtitle" = "Guests: %@, Services: %@, Read receipts: %@";
+"conversation.create.options.subtitle.without_services" = "Guests: %@, Read receipts: %@";
```

Comment:
**Note:** After merging `Navigation overhaul`, we don't need it because the conversation creation UI has been changed.